src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -97,7 +134,29 @@ private void submitTopicOperation(PathOperation op, OpType opType) {
     }
   }
 
-  private void submitTopicCreateOperation() {}
+  private void submitTopicCreateOperation(final PathOperation op) {
+
+    final String topicName = op.item().name();
+    try {
+      val newTopic = new NewTopic(topicName, 1, (short) 1);
+      val createTopicsResult = adminClient.createTopics(Collections.singleton(newTopic));
+      val createTopicFuture = createTopicsResult.values().get(topicName).
+              whenComplete(kafkaTopicFuture(topicName, op));
```

Comment:
please try using something like this:
`whenComplete((result, thrown) -> handleTopicCreate(op, topicName, thrown));`

Reply:
changed